### ISSUE TITLE
Add contributing guidelines, issue/PR templates

### ISSUE THREAD

**Issue by Panquesito7** [MinetestForFun/server-minetestforfun#541]
What do you think about adding issue/PR templates, and/or contributing guidelines?
Post your feedback here. 🙂

**Comment by Darcidride** [comment]
I agree, it will be a great thing to do, some templates are already available, we just need to pick one
